README.md

```diff
@@ -281,6 +281,33 @@ test them on a copy of your database, not on the live database right away.
 In a production setup, you should also have regular, automated database
 backups set up.
 
+
+## Deployment
+
+### Docker
+
+To run the app, you can:
+
+    $ docker build -t healthchecks:latest .
+
+    $ docker volume create \
```

Comment:
Necessary. You can simply mount ``/data`` to a volume.

Reply:
yes